src/client/components/pages/collection.js

```diff
@@ -283,8 +284,9 @@ class CollectionPage extends React.Component {
 							<Button
 								bsSize="small"
 								bsStyle="danger"
-								className="margin-bottom-d5"
+								className="margin-right-d5"
 								disabled={!this.state.selectedEntities.length}
+								style={{marginRight: '0.5rem'}}
```

Comment:
The style can be removed, since we're using the class `margin-right-d5` instead.